### PR TITLE
DBG_ENABLE #267

### PR DESCRIPTION
**Pull request by gregw**
DBG_ENABLE cleanup for #267 

This uses case insensitive booleans for both ALPN_ENABLE and DBG_ENABLE

If the booleans are not set, then ALPN_ENABLE defaults to false and DBG_ENABLE defaults to true, unless CDBG_DISABLE is set to non null.

I'm not sure we want to defer to the CDBG_DISABLE in that way?

### CODE REVIEW

File: openjdk8/src/main/docker/setup-env.bash [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
 ALPN_BOOT=
-if [[ -n "$ALPN_ENABLE" ]]; then
+if isTrue "${ALPN_ENABLE:=false}" ; then
   ALPN_BOOT="$( /opt/alpn/format-env-appengine-vm.sh )"
 fi
 
 DBG_AGENT=
-if [[ "$GAE_PARTITION" = "dev" ]]; then
-  if [[ -n "$DBG_ENABLE" ]]; then
+if isTrue "${DBG_ENABLE:=$( if [[ -z ${CDBG_DISABLE} ]] ; then echo true; else echo false ; fi )}" ; then
```
Comment (aozarov) [inline openjdk8/src/main/docker/setup-env.bash]: Fine as is but do you really think we should also consider CDBG_DISABLE as a fallback to DBG_ENABLE?
From the user perspective I thought we said that we only document and consider  DBG_ENABLE (and then
manipulate the other "internal" variables" based on it, as you already do).

Comment (gregw) [inline openjdk8/src/main/docker/setup-env.bash]: I'm 50:50 on this.  I can see that it is a good backwards compatibility fallback... but then I can't find any documentation about it either... other than the code in the cdbg script.   Although I can imagine somebody looking at that script and then being driven crazy setting CDBG_DISABLE and wondering why it is not disabled.

Note that even if we don't respect it, we still have to unset it in our script, so we can't totally remove handling for CDBG_DISABLE.  If we have to code for it, then why not respect it?    Hmmm think I'm talking myself to 60:40 keeping it.

But its OK either way.  your call!

Comment (aozarov) [inline openjdk8/src/main/docker/setup-env.bash]: Ok, lets leave it but not document it.

**Comment by aozarov** [comment]
Do you mind to include the doc change for this script change in this PR?

**Comment by gregw** [comment]
@aozarov I've updated the `README.md` of openjdk8 and jetty9, but the `README.md` for jetty-9-compat is woefully inadequate and also on the base rather than the image.  I'll open another issue for that.

**Pull request merged**